### ISSUE TITLE
Summary view does not work for wide SFrame

### ISSUE THREAD

**Issue by afranklin** [apple/turicreate#700]
I don’t see tooltips, it freezes on me when I have more columns. On this large random SFrame, I get stuff cut off

In [8]: sf
Out[8]:
Columns:
	col_int.0	int
	col_int.1	int
	col_int.2	int
	col_int.3	int
	col_int.4	int
	col_int.5	int
	col_int.6	int
	col_int.7	int
	col_int.8	int
	col_int.9	int

Rows: 5120000000

Data:
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
| col_int.0 | col_int.1 | col_int.2 | col_int.3 | col_int.4 | col_int.5 | col_int.6 |
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
|     21    |     11    |     9     |     26    |     14    |     21    |     13    |
|     9     |     23    |     10    |     12    |     13    |     26    |     2     |
|     20    |     10    |     20    |     2     |     16    |     16    |     18    |
|     26    |     6     |     10    |     14    |     11    |     19    |     11    |
|     12    |     10    |     15    |     8     |     7     |     7     |     14    |
|     11    |     4     |     7     |     10    |     18    |     6     |     7     |
|     12    |     12    |     6     |     13    |     10    |     13    |     9     |
|     18    |     17    |     9     |     16    |     15    |     6     |     12    |
|     4     |     14    |     12    |     10    |     22    |     11    |     15    |
|     5     |     22    |     27    |     18    |     11    |     6     |     8     |
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
+-----------+-----------+-----------+
| col_int.7 | col_int.8 | col_int.9 |
+-----------+-----------+-----------+
|     8     |     6     |     4     |
|     11    |     12    |     27    |
|     6     |     7     |     21    |
|     20    |     7     |     18    |
|     11    |     15    |     10    |
|     13    |     7     |     17    |
|     22    |     27    |     6     |
|     20    |     12    |     13    |
|     23    |     23    |     4     |
|     16    |     7     |     24    |
+-----------+-----------+-----------+
[5120000000 rows x 10 columns]
Note: Only the head of the SFrame is printed.
You can use print_rows(num_rows=m, num_columns=n) to print more rows and columns.

**Comment by afranklin** [comment]
Case that will still repro is 100+ columns. For now we error out at ~50.

**Comment by abhishekpratapa** [comment]
Can I ask to clarify the issue? Just curious because Are we saying the number of rows is the issue or the number of column is the issue? Or both. I'm assuming columns.

**Comment by abhishekpratapa** [comment]
```
Materializing SFrame
Warning: Skipping column 'X51' [integer]. Unable to show more than 50 columns.
Further warnings of more than 50 columns will be suppressed.
```

**Comment by TobyRoseman** [comment]
This seems like it fixed in TuriCreate 6.4. Below is how a similar SFrame get displayed in 6.4:
```
Columns:
	col_int.0	int
	col_int.1	int
	col_int.2	int
	col_int.3	int
	col_int.4	int
	col_int.5	int
	col_int.6	int
	col_int.7	int
	col_int.8	int
	col_int.9	int

Rows: 99

Data:
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
| col_int.0 | col_int.1 | col_int.2 | col_int.3 | col_int.4 | col_int.5 | col_int.6 |
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
|     83    |     82    |     58    |     10    |     13    |     82    |     19    |
|     55    |     43    |     2     |     13    |     25    |     25    |     81    |
|     85    |     58    |     56    |     17    |     53    |     28    |     20    |
|     12    |     79    |     72    |     98    |     85    |     21    |     17    |
|     66    |     96    |     59    |     39    |     60    |     16    |     31    |
|     87    |     71    |     90    |     12    |     86    |     88    |     57    |
|     43    |     98    |     76    |     84    |     72    |     31    |     88    |
|     97    |     68    |     40    |     27    |     0     |     35    |     2     |
|     72    |     50    |     33    |     22    |     47    |     41    |     5     |
|     19    |     92    |     88    |     9     |     12    |     45    |     41    |
+-----------+-----------+-----------+-----------+-----------+-----------+-----------+
+-----------+-----------+-----------+
| col_int.7 | col_int.8 | col_int.9 |
+-----------+-----------+-----------+
|     25    |     46    |     5     |
|     54    |     39    |     82    |
|     97    |     93    |     70    |
|     31    |     74    |     85    |
|     30    |     33    |     49    |
|     84    |     95    |     12    |
|     8     |     18    |     46    |
|     2     |     51    |     84    |
|     74    |     82    |     74    |
|     66    |     4     |     94    |
+-----------+-----------+-----------+
[99 rows x 10 columns]
Note: Only the head of the SFrame is printed.
You can use print_rows(num_rows=m, num_columns=n) to print more rows and columns.
```
Looks good to me.
